.github/workflows/qa.yaml

```diff
@@ -10,7 +10,7 @@ on:
 env:
   apt_dependencies: >-
     ca-certificates curl dconf-cli gcc gettext git libnss-wrapper libsmbclient-dev
-    libwbclient-dev pkg-config python3-coverage samba sudo
+    libkrb5-dev libwbclient-dev pkg-config python3-coverage samba sudo
```

Comment:
shouldn’t we add it explicitely as a build-dep in debian/control too?

Reply:
Yeah, no black magic from debhelper here, it needs to be explicit (well, it’s probably pulled by sss -dev package). But as we rely on it, let’s ensure this is explicit.